web/classroom_views.py

```
@@ -0,0 +1,101 @@
+import json
+from django.shortcuts import render, get_object_or_404
+from django.http import JsonResponse
+from django.contrib.auth.decorators import login_required
+from django.views.decorators.http import require_http_methods, require_POST
+from .models import ClassroomGameState, ChairOccupancy
+
+
+@login_required
+def virtual_classroom(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `virtual_classroom`

(ANN201)